hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/rollback/SparkMarkerBasedRollbackStrategy.java

```diff
@@ -91,4 +97,37 @@ public SparkMarkerBasedRollbackStrategy(HoodieTable<T, JavaRDD<HoodieRecord<T>>,
         FSUtils.getPartitionPath(config.getBasePath(), partitionPathStr), fileId, HoodieFileFormat.HOODIE_LOG.getFileExtension(), baseCommitTime)
         .collect(Collectors.toMap(HoodieLogFile::getFileStatus, value -> value.getFileStatus().getLen()));
   }
+
+  /**
+   * Gets all marker paths.
+   *
+   * @param instant     instant of interest to rollback
+   * @param parallelism parallelism to use
+   * @return a list of all markers
+   * @throws IOException
+   */
+  private List<String> getAllMarkerPaths(String instant, int parallelism) throws IOException {
+    String markerDir = table.getMetaClient().getMarkerFolderPath(instant);
+    FileSystem fileSystem = FSUtils.getFs(markerDir, context.getHadoopConf().newCopy());
```

Comment:
you can always do table.getMetaClient().getFs() to get this object. its more standard across teh code.

Reply:
Right.  Fixed.